<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FScreen</class>
 <widget class="QWidget" name="FScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2566</width>
    <height>1023</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QLabel" name="frame1">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>640</width>
     <height>512</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="frame2">
   <property name="geometry">
    <rect>
     <x>642</x>
     <y>1</y>
     <width>640</width>
     <height>512</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="frame3">
   <property name="geometry">
    <rect>
     <x>1283</x>
     <y>1</y>
     <width>640</width>
     <height>512</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="frame4">
   <property name="geometry">
    <rect>
     <x>1924</x>
     <y>1</y>
     <width>640</width>
     <height>512</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="frame5">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>514</y>
     <width>640</width>
     <height>512</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="frame6">
   <property name="geometry">
    <rect>
     <x>642</x>
     <y>514</y>
     <width>640</width>
     <height>512</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="frame7">
   <property name="geometry">
    <rect>
     <x>1283</x>
     <y>514</y>
     <width>640</width>
     <height>512</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="frame8">
   <property name="geometry">
    <rect>
     <x>1924</x>
     <y>514</y>
     <width>640</width>
     <height>512</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>quit</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
